### ISSUE TITLE
Comments are not being emailed to all project members

### ISSUE THREAD

**Issue by oraclerob** [upstreamplugin/UpStream-Email-Notifications#62]
If a team member for a project makes a comment, only the owner and the author are receiving an email update. I think all team members for the project should receive and email.

**Comment by stevejburge** [comment]
@oraclerob I agree. It's something that's on our radar: https://upstreamplugin.com/upstream-updates-in-may-plus-improved-notifications/

**Comment by oraclerob** [comment]
If you give me access to the repo I can have a quick look to see if I can add it in quickly.

**Comment by stevejburge** [comment]
Sure, there's a probably a short-term fix here. Then in the longer term, we can add the more flexible library

**Comment by oraclerob** [comment]
Thats what I was thinking - short term fix initially then your more advanced preview screen later adding all the bells and whistles. The confusion is currently task and milestones email everyone but the comments don't.

**Comment by stevejburge** [comment]
Makes sense to me. Thanks for catching the iconsistency
